test/muuntaja/core_perf_test.clj

```diff
@@ -435,72 +437,74 @@
     (cc/bench (app (request!)))
     (cc/bench (ring-stream! (app (request!))))))
 
-;; file sizes about about the size. good enough.
-(defn e2e-json-comparison-different-payloads []
-  (doseq [file ["dev-resources/json10b.json"
-                "dev-resources/json100b.json"
-                "dev-resources/json1k.json"
-                "dev-resources/json10k.json"
-                "dev-resources/json100k.json"]
-          :let [data (cheshire/parse-string (slurp file))
-                request (json-request data)
-                request! (request-stream request)]]
+(defmacro bench [& body]
+  `(let [result# (cc/benchmark ~@body {})
+         mean# (-> result# :mean first (* 1000000000) long)]
+     (cc/report-result result#)
+     mean#))
```

Comment:
So I guess my alternative proposal would be to store the full results of `cc/benchmark` instead of just the mean. If we're going to archive the results, it would be nice to be able to look at all the data we got.